test/e2e_node/gpu_device_plugin.go

```diff
@@ -65,19 +69,33 @@ var _ = framework.KubeDescribe("NVIDIA GPU Device Plugin [Feature:GPUDevicePlugi
 			if framework.NumberOfNVIDIAGPUs(getLocalNode(f)) < 2 {
 				Skip("Not enough GPUs to execute this test (at least two needed)")
 			}
+
+			By("Adding extended resource not managed by device plugin")
+			patch := []byte(fmt.Sprintf(`[{"op": "add", "path": "/status/capacity/%s", "value": "5"}]`, escapeForJSONPatch(nonDeviceResource)))
+			f.ClientSet.CoreV1().RESTClient().Patch(types.JSONPatchType).Resource("nodes").Name(framework.TestContext.NodeName).SubResource("status").Body(patch).Do()
```

Comment:
Since this is a serial test, it might be cleaner to just do this (although it uses an additional request, you avoid writing a patch in a raw string):
```
node, err := f.ClientSet.CoreV1().Nodes().Get(framework.TestContext.NodeName, metav1.GetOptions{})
if err != nil {
	return err
}
node.Status.Capacity[nonDeviceResource] = resource.MustParse("5")
_, err = f.ClientSet.CoreV1().Nodes().Update(node)
if err != nil {
	return err
}
```

Reply:
Thanks a lot for the tip! This is much cleaner. Updated as suggested.